modular_skyrat/modules/sec_haul/code/misc/vending.dm

```diff
@@ -30,9 +30,6 @@
 		/obj/item/watertank/pepperspray = 2,
 		/obj/item/storage/belt/holster/energy = 4,
 		/obj/item/storage/box/holobadge = 1,
-		/obj/item/ammo_box/magazine/m9mm/rubber = 12,
-		/obj/item/ammo_box/magazine/m9mm/ihdf = 12,
-		/obj/item/ammo_box/c10mm/speedloader/rubber = 12,
 		/obj/item/ammo_box/advanced/s12gauge/frangible = 2,
```

Comment:
Don't we usually have murphy magazines here now, or how is that currently done?

Reply:
We do, I'll resolve to master

Reply:
Resolved to master